docs/wiki/User Guide/Configurations.md

```diff
@@ -167,7 +167,7 @@ The following configurations are inherited from the open source Kafka client con
 | metric.anomaly.class	                            | Class     | N	        | com.linkedin.kafka.cruisecontrol.detector.KafkaMetricAnomaly	        | The name of class that extends metric anomaly.	                                                                                                                                                                                                                                    |
 | self.healing.goals	                            | List	    | N	        | []	                                                                | The list of goals to be used for self-healing relevant anomalies. If empty, uses the default.goals for self healing.	                                                                                                                                                                |
 | anomaly.notifier.class                            | Class     | N         | com.linkedin.kafka.cruisecontrol.detector.notifier.NoopNotifier       | The notifier class to trigger an alert when an anomaly is violated.                                                                                                                                                                                                                   |
-| anomaly.detection.goals                           | List      | N         | com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal    | The goals that anomaly detector should detect if they are violated.                                                                                   |
+| anomaly.detection.goals                           | List      | N         | com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderReplicaPerLiveBrokerGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal    | The goals that anomaly detector should detect if they are violated.    |
```

Comment:
`LeaderReplicaPerLiveBrokerGoal` is an incorrect update -- it is a goal that does not exist.
Please use correct names in our documentation.